crates/matrix-sdk/src/client/builder.rs

```diff
@@ -453,9 +454,9 @@ impl ClientBuilder {
         // only uploaded to a OpenTelemetry collector if the span gets dropped.
         // We still want it around so other methods that get called by this
         // client instance are connected to it, so we clone.
-        drop(self.root_span.clone());
+        // drop(self.root_span.clone());
```

Comment:
cc @poljar, I think you said this doesn't work anyways? Maybe we should just remove it then, instead of commenting it out.

Reply:
I was investigating this part just now. I don't know what to do with this.

Reply:
Done.